lib/tasks/wayback.rake

```diff
@@ -0,0 +1,35 @@
+desc 'Request the Wayback Machine to snapshot a specific URL or all new '\
+  'article URLs for the past 24 hours if no argument is given'
+
+task :wayback, [:url] => :environment do |_t, args|
+  app = JSON.parse(File.read(Rails.root.join('app.json')))
+  api = 'https://pragma.archivelab.org/'
+  urls = []
+
+  if args[:url]
+    urls.append args[:url]
+  else
+    since = 1.day.ago.to_s
+    Article.where("updated_at > '#{since}'").each do |article|
+      urls.append [app['website'], article.path].join('')
+    end
+  end
+
+  uri = URI.parse(api)
+  urls.each do |url|
```

Comment:
is there a rate limit?

I guess there won't be _that_ many urls, so maybe no need to worry about an accidental ddos

Reply:
I have a rate limit before but I was blasting through every single article on the site. I can't imagine that we'll run into it with just today's articles.